api/utils.py

```diff
@@ -8,7 +8,7 @@
 
 class APIUtils():
     QUERY_TERMS = [
-        'keyword', 'title', 'author', 'subject', 'viaf', 'lcnaf',
+        'keyword', 'title', 'author', 'subject', 'lccn', 'isbn', 'viaf', 'lcnaf',
```

Comment:
If we don't extend `authorityQuery` we can drop this update

Reply:
I agree in not extending the authorityQuery and instead making a new method.

Reply:
So I'll delete `lccn` and `isbn`